docker-compose.single.yaml

```diff
@@ -0,0 +1,67 @@
+version: "3.7"
+
+services:
+  validator-node:
```

Comment:
Name this `single-validator` and move it to the main docker-compose config. @evilpeach please help review.